LICENSE.md

```diff
@@ -1,7 +1,5 @@
 MIT License
 
-Copyright (c) 2017 Hashrocket
```

Comment:
@plicjo tsk tsk tsk, you can't modify the license, only sublicense

![tsk tsk tskt](https://media.tenor.com/images/a754d60ae8d2c72d480db406e9405acf/tenor.gif)

Reply:
Got you 💪

Reply:
I just added it back